src/features/auth/ui/step-profile.tsx

```
@@ -14,11 +14,10 @@ import {
   SelectValue,
 } from '@/shared/ui/common/select'
 import { cn } from '@/shared/utils'
-import type { RegisterFormValues } from '../lib'
 import { profileStepData } from '../model'
 
 interface StepProfileProps {
-  form: UseFormReturn<RegisterFormValues>
+  form: UseFormReturn<any>
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)